ray-operator/config/samples/ray-service.tpu-multi-host.yaml

```diff
@@ -72,15 +72,15 @@ spec:
             image: rayproject/ray:2.9.0-py310
             resources:
               limits:
-                cpu: "1"
+                cpu: "100"
```

Comment:
typo to use 100 CPUs?

Reply:
[0ccb431](https://github.com/ray-project/kuberay/pull/2198/commits/0ccb4318d630f2d71243de8e3725f29562c5e7a8)